app/javascript/components/froggo-dropdown.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="relative" v-on-clickaway="closeDropdown">
+    <div class="cursor-pointer" @click="showBody = !showBody">
+      <slot name="btn">
+        Open me
+      </slot>
+    </div>
+    <div class="absolute z-10 right-0"  v-if="showBody">
+      <slot name="body" :close-dropdown="closeDropdown">
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>':close-dropdown' should be on a new line  vue/max-attributes-per-line